techniques/system/common/1.0/update.st

```diff
@@ -268,6 +267,7 @@ bundle agent update_action
         file_select       => exclude("${rudder_promises_timestamp_tmp}"),
         classes           => success("config", "no_update", "config_ok");
 
+&if(INITIAL)&
```

Comment:
I think this should be "*not* INITIAL", otherwise we won't update ncf in the main promises anymore!